supertokens_python/recipe/passwordless/utils.py

```diff
@@ -126,15 +113,18 @@ def __init__(self,
 
 class ContactEmailOnlyConfig(ContactConfig):
     def __init__(self,
-                 create_and_send_custom_email: Callable[
-                     [CreateAndSendCustomEmailParameters, Dict[str, Any]], Awaitable[None]],
+                 create_and_send_custom_email: Union[Callable[
+                     [CreateAndSendCustomEmailParameters, Dict[str, Any]], Awaitable[None]], None] = None,
                  validate_email_address: Union[Callable[[
-                     str], Awaitable[Union[str, None]]], None] = None
+                     str], Awaitable[Union[str, None]]], None] = None,
+                 email_delivery: Union[EmailDeliveryConfig[TypePasswordlessEmailDeliveryInput], None] = None
```

Comment:
This should not be tied to a contact method

Reply:
Didn't get you.

Reply:
Fixed.